packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -1104,6 +1104,24 @@ class ListenableBuilder extends AnimatedWidget {
     this.child,
   });
 
+  /// Creates a [ListenableBuilder] that responds to changes in multiple [listenables].
+  ///
+  /// This constructor is useful when you need to react to changes from multiple
+  /// [Listenable] objects simultaneously.
+  ///
+  /// [listenables] is an iterable of [Listenable] objects whose changes will trigger rebuilds.
```

Comment:
Sentences should start grammatically correct with capital letters, see https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-correct-grammar